src/core/core/rpc_engine.cpp

```diff
@@ -144,7 +144,12 @@ namespace dsn {
                 req->server_address.group_address()->leader_forward();
             }
 
-            call->set_delay(delay_ms);
+            // the "delay_ms" is simulator's net_delay_milliseconds,
+            // the "call->delay_milliseconds()" is user specified delay time to dispatch response task,
```

Comment:
As in you way, the value set by response_task->set_delay() will always be
over-writed?

On Thu, Jul 28, 2016 at 4:41 PM, Zhenyu Guo <notifications@github.com>
wrote:

> In src/core/core/rpc_engine.cpp
> <https://github.com/imzhenyu/rDSN/pull/497#discussion_r72584448>:
>
> > @@ -144,7 +144,12 @@ namespace dsn {
> >                  req->server_address.group_address()->leader_forward();
> >              }
> >
> > -            call->set_delay(delay_ms);
> > +            // the "delay_ms" is simulator's net_delay_milliseconds,
> > +            // the "call->delay_milliseconds()" is user specified delay time to dispatch response task,
>
> I think in the earliest time, it should be call->set_delay(delay_ms);,
> which is easy.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/imzhenyu/rDSN/pull/497/files/d8635e3fc4fb2e12960d1e592fce66a24bfaedea#r72584448>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAgQPax32IKiycol17hiFhMxSstd9q40ks5qaGtOgaJpZM4JW8NK>
> .
>